### PR TITLE
Ensure pre and post ceph deployments have different names

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

**Review by slagle — COMMENTED** [review]
+2

**Comment by slagle** [comment]
/hold
for beta freeze

File: examples/post_ceph_hci/values.yaml [hunk 1]
```diff
@@ -156,6 +156,7 @@ data:
         - name: tenant
           subnetName: subnet1
   deployment:
+    name: edpm-deployment-ipam-post-ceph
```
Comment (SeanMooney) [inline examples/post_ceph_hci/values.yaml:159]: im not entirely sure why ipam is in the name but otehrwas this change looks good to me

**Review by SeanMooney — APPROVED** [review]

**Pull request merged**